opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -16,8 +16,7 @@
   OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include <opm/input/eclipse/EclipseState/Grid/FieldProps.hpp>
```

Comment:
We generally try to place the `.cpp` file's own header first in order to ensure that the header can be compiled in isolation.  Please put this include statement back to its original location.

Reply:
I actually thought I was doing you a favor as you are the one that is often insisting on sorted header includes. Quite confusing.,,,

Reply:
Changed back